### PR TITLE
Update hubploy for a bugfix

### PR DESCRIPTION
**Pull request by consideRatio**
See https://github.com/yuvipanda/hubploy/pull/68, ping @scottyhq - I think this doesn't solve [the issue you had](https://app.circleci.com/pipelines/github/pangeo-data/pangeo-cloud-federation/1078/workflows/6a4b0e8b-a360-4155-9f6b-2d3d17147053/jobs/1223), but I'm not sure it doesn't either :p

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Hmm any idea if the CI failure is from our existing issues or from updating hubploy?

```
Traceback (most recent call last):

  File "/home/circleci/repo/venv/bin/hubploy", line 11, in <module>

    sys.exit(main())

  File "/home/circleci/repo/venv/lib/python3.7/site-packages/hubploy/__main__.py", line 99, in main

    with auth.registry_auth(args.deployment, args.push, args.check_registry):

  File "/usr/local/lib/python3.7/contextlib.py", line 114, in __enter__

    raise RuntimeError("generator didn't yield") from None

RuntimeError: generator didn't yield

```

**Comment by consideRatio** [comment]
I hade to run before i could figure this out, but im clueless atm :/

**Comment by salvis2** [comment]
That error is probably a `hubploy` thing.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/yuvipanda/hubploy@d619b2da6cae00c1b14b05b176fb66a5413eb01f
+git+https://github.com/yuvipanda/hubploy@6e0ecb936e872fe57cd85fc1a1807f0f230a249d
```
Comment (consideRatio) [inline requirements.txt]:
```suggestion
git+https://github.com/yuvipanda/hubploy@a01a6dad8122e84d54702390d02d1f3836adef77
```

**Comment by consideRatio** [comment]
This is ready for merge!

**Comment by salvis2** [comment]
Thanks @consideRatio !

**Pull request merged**